search/query.go

```diff
@@ -27,6 +30,10 @@ const (
 	// Boost for exact phrase match, without slop.
 	EXACT_BOOST = 1.5
 
+	SPAN_NEAR_BOOST = 0.01
```

Comment:
Why so low?
Why not disable span at all?
Please comment that constant and it's low values.
Please add in the comment example of results of why you did this.

This is important because we forget all those examples with time.

Reply:
TBD add comment example (near the function)